diff --git a/Cargo.toml b/Cargo.toml
index a79cad4..72cace0 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -28,8 +28,10 @@ colored = { version = "2.0.0" }
 rusb = { version = "0.9.1", optional = true }
 libusb1-sys = { version = "0.6.2", optional = true }
 
+[patch.crates-io]
+rusty_libimobiledevice = { git = 'https://github.com/xunmod/rusty_libimobiledevice' }
+
 [features]
-default = ["static", "vendored"]
 static = ["rusty_libimobiledevice/static", "plist_plus/static"]
 usb = ["libusb1-sys", "rusb"]
 vendored = ["rusty_libimobiledevice/vendored", "plist_plus/vendored"]
