diff --git a/src/devices.rs b/src/devices.rs
index cce83bf..0d54065 100644
--- a/src/devices.rs
+++ b/src/devices.rs
@@ -42,6 +42,7 @@ impl SharedDevices {
             match std::env::consts::OS {
                 "macos" => "/var/db/lockdown",
                 "linux" => "/var/lib/lockdown",
+                "android" => "@TERMUX_PREFIX@/var/lib/lockdown",
                 "windows" => "C:/ProgramData/Apple/Lockdown",
                 _ => panic!("Unsupported OS, specify a path"),
             }
